src/explore-education-statistics-common/src/modules/charts/components/LineChartBlock.tsx

```diff
@@ -39,6 +39,7 @@ import {
   YAxis,
 } from 'recharts';
 import getDataSetCategoryConfigs from '@common/modules/charts/util/getDataSetCategoryConfigs';
+import CustomReferenceLineLabel from '@common/modules/charts/components/CustomReferenceLineLabel';
```

Comment:
Don't we need these for vertical/horizontal bar blocks too?

Reply:
This solution doesn't really work for them, it looks wrong with the width of the bars.  I've checked with Laura and she's happy to leave the bar charts as they are now, with the labels centre aligned, as the bars provide a bit of padding which makes the text less likely to overflow the chart and reference lines are used less on them.

Reply:
Cool, good enough for me!